src/test/regress/specs/isolation_data_migration.spec

```diff
@@ -0,0 +1,73 @@
+setup
+{
+    CREATE TABLE migration_table (test_id integer NOT NULL, data text);
+}
+
+teardown
+{
+    DROP TABLE migration_table;
+}
+
+session "s1"
+
+step "s1-begin"
+{
+    BEGIN;
+}
+
+step "s1-begin-serializable"
+{
+    BEGIN TRANSACTION ISOLATION LEVEL SERIALIZABLE;
+	SELECT 1;
```

Comment:
We don't get a snapshot until the first statement, so if we don't do anything before then, we'll see all preceding writes regardless of whether we're in serializable mode or not. If we're in serializable mode, the second statement will normally see the same copy of the database as the first statement. In read committed mode, we'd writes that have been committed since.